druid-shell/examples/empty_window.rs

```diff
@@ -0,0 +1,136 @@
+// Copyright 2018 The Druid Authors.
```

Comment:
Do we need this example? It is very similar to shello example

Reply:
nope not really. will purge it.